example/components/test_ping.tavern.yaml

```diff
@@ -0,0 +1,30 @@
+---
+
+test_name: Test authenticated /ping
+
+includes:
+  - !include common.yaml
+  - !include components/auth_stage.yaml
+
+stages:
+  - name: Unauthenticated /ping
+    request:
+      url: "{service:s}/ping"
+      method: GET
+    response:
+      status_code: 401
+  - type: ref
+    id: login_get_token
```

Comment:
I don't have a problem with making the change, but 2 things to consider:
1) There already is a ```name```. Perhaps ```stage_id```?
2) These are just additional attributes of the a stage. Nothing else seems to be prefixed with ```stage_```.

Let me know if you still want them (a) prefixed and (b) renamed. Should be able to get that in quickly.

Reply:
I think that having 'id' (prefixed or otherwise) is just confusing because the stages are referred to by a 'name', not an 'id'.

If I was reading the documentation and it said "`stage_id` should be the name of a stage" I would wonder why they didn't just call it `stage_name`, if you catch my drift. Perhaps `stage_name_ref`?

I agree about the 'type' though, having that as 'stage_ref' would be a bit uneccesarily verbose

Reply:
In between intially looking at this and making that comment I forgot about the stage 'id', my mistake